staking-miner-playground/runtime/build.rs

```diff
@@ -1,5 +1,9 @@
 use substrate_wasm_builder::WasmBuilder;
 
+pub const NOMINATORS: Option<&str> = option_env!("N");
+pub const CANDIDATES: Option<&str> = option_env!("C");
+pub const VALIDATORS: Option<&str> = option_env!("V");
```

Comment:
So, the N, C and V env vars can be set _at compile time_ to determine the value of these things, and we don't mind that we can't configure them at runtime?

Reply:
yupp, that's the best option right now :(

Reply:
interesting, are these constants then accessible at runtime for reading?

Reply:
Just to check; if Subxt _could_ read the constants from the node, would that help?

(I know it's not something we normally need to do, but I geuss I'm not feeling too opposed to the idea that when you read a constant, you can opt to load it from the node rather than the metadata; for instance you can make the client use older metadata or not subscribe to runtime updates or whatever, so this would be a good "fallback" option to provide anyway IMO)

Reply:
(update: as far as we can see, there's no way to read newer constants from a node, so alas :))